integration_tests/ethereum.go

```diff
@@ -23,3 +35,175 @@ type EthereumGenesis struct {
 	Config     EthereumConfig        `json:"config"`
 	Alloc      map[string]Allocation `json:"alloc"`
 }
+
+const approveERC20ABIJSON = `
```

Comment:
Is all of this being ported from one of the other repo's integration tests?

Reply:
I think I pulled this from gravity yeah